guidelines/application_requirements.md

```diff
@@ -93,7 +93,7 @@ This document is not intended to cover SDKs (e.g. fairing) or client side tools
 ## Reference
 
 * TFJob and PyTorch 1.0 [exit criterion](http://bit.ly/operators-exit-criterion)
-* [CNCF Graduation Criteria](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc)
+* [CNCF Graduation Criteria](https://github.com/cncf/toc/blob/main/process/graduation_criteria.md)
```

Comment:
Shouldn't we point to https://github.com/cncf/toc/blob/main/.github/ISSUE_TEMPLATE/template-graduation-application.md instead?

Reply:
yes, you are right. udpating the PR